cli/js/fetch_test.ts

```diff
@@ -160,6 +166,7 @@ testPerm({ net: true }, async function fetchWithRedirection(): Promise<void> {
   assert(body.includes("<title>Directory listing for /</title>"));
 });
 
+// TODO: leaking resources
 testPerm({ net: true }, async function fetchWithRelativeRedirection(): Promise<
```

Comment:
Following `fetch` tests that are disabled have pretty complex problem with leaking `httpBody` resource - this needs to be fixed